src/support/timefuncs.c

```diff
@@ -7,7 +7,7 @@
 #if defined(_OS_WINDOWS_)
 #include <sys/timeb.h>
 #else
-#include <sys/time.h>
+#include <time.h>
```

Comment:
Not 100% sure about this, but was required to get it to build on my system. CI will tell if this was a mistake :shrug: